src/benchmark.cc

```diff
@@ -446,7 +446,11 @@ void PrintUsageAndExit() {
           "          [--benchmark_out_format=<json|console|csv>]\n"
           "          [--benchmark_color={auto|true|false}]\n"
           "          [--benchmark_counters_tabular={true|false}]\n"
-          "          [--v=<verbosity>]\n");
+#if defined HAVE_LIBPFM
```

Comment:
I'd propose separating this out of this PR.

That aside, I wouldn't condition this on HAVE_LIBPFM, for 2 reasons:

* while currently the case, !HAVE_LIBPFM isn't equivalent to perf counters not supported; for instance, if someone implements it on windows (libpfm is a linux thing).
* the flag is always available, just doesn't function if perflab support isn't there.

My preference would be to add an extra note after the format of the option if !PerfCounters::kSupported, like for example before `exit(0)`

```
if (!PerfCounters::kSupported) fprintf(stdout, "NOTE: --benchmark_perf_counters not supported on this platform/build. See documentation for details");
```

Reply:
Yes. Now I'm thinking about both build time support and runtime support. This is definitely going in it's own separate direction. I'd better not include it in this PR.